app/javascript/components/editor/TestsGroupList.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { TestStatus, Test } from './types'
+import { GraphicalIcon } from '../common/GraphicalIcon'
+import { TestsGroup } from './TestsGroup'
+
+export function TestsGroupList({ tests }: { tests: Test[] }): JSX.Element {
+  const testsWithIndex = tests.map((test, i) => ({ index: i + 1, ...test }))
+  const firstFailedTestIdx = testsWithIndex.findIndex(
```

Comment:
I think you can use [`some()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some), which returns a `boolean`:

```suggestion
  const firstFailedTestIdx = testsWithIndex.some(
```

Reply:
True, but I think further down firstFailedTestIdx is used

Reply:
Yes, we use an ID here. I've rewritten the code for it to be clearer now.